README.md

```diff
@@ -250,7 +250,6 @@ twine upload dist/*
 - Implement auditing for the remaining "special" QuerySet write operations:
   - `bulk_create()`
   - `bulk_update()`
-  - `update()`
```

Comment:
[This block](https://github.com/dimagi/django-field-audit/blob/main/README.md?plain=1#L127-L132) also needs updated.

Reply:
See 866af4c7bb877f02271a50cc55b9a7aadf9857c1 and 6497346678c6e36322d9f511335e27dbaf77e461 🤦🏻‍♂️